src/jit/error.h

```diff
@@ -80,7 +80,21 @@ extern void noWayAssertBodyConditional(
     );
 extern void noWayAssertBodyConditional(const char* cond, const char* file, unsigned line);
 
+// Define MEASURE_NOWAY to 1 to enable code to count and rank individual noway_assert calls by occurrence.
+// These asserts would be dynamically executed, but not necessarily fail. The provides some insight into
```

Comment:
This key point (that you're measuring how many are executed, not how many fail) was one I missed at first. It might be useful to mention that in the `MEASURE_NOWAY` section in compiler.cpp as well.